stonesoup/initiator/simple.py

```diff
@@ -74,6 +74,9 @@ class SimpleMeasurementInitiator(GaussianInitiator):
     prior_state = Property(GaussianState, doc="Prior state information")
     measurement_model = Property(MeasurementModel, doc="Measurement model")
     skip_non_reversible = Property(bool, default=False)
+    diag_load = Property(float,
```

Comment:
I think a short sentence is main doc string above on why you'd want a diagonal load be useful.

Reply:
Should this be in the docstring of the diag_load Property or the docstring of the class?

Reply:
For the class I think. Let's you be more verbose.

Reply:
Done. Note - I added a check to make sure that diag_load >= 0 in the __init__().